instrumentation_wrapper.cpp

```diff
@@ -241,10 +244,11 @@ void instrument(
 		if(ocnt != OLEN-1)  ocnt++;
 		else {
 			clock_t  ts0;
-			if(!timestamps.read_nb(ts0))  timestamp_unf = true;
+			if(!timestamps.read_nb(ts0))  timestamp_unf |= true;
```

Comment:
Why should this original plain assignment be described as a read-modify-write operation?

Reply:
It doesnt need to be. I previously noticed that the assignment for overflow and underflow flags was different and I was concerned the underflow flag wouldn't persist across multiple processed feature maps and thus the host would miss it. But looking at it again I realise `timestamp_unf` is never set to false, and thus it will persist.

I removed that commit from the PR. Thanks for spotting it.